### ISSUE TITLE
"Auth is not properly configured" error preventing signup/in

### ISSUE THREAD

**Issue by mtliendo** [aws-amplify/amplify-js#9937]
### Before opening, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-js/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-js/discussions).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### JavaScript Framework

React, Next.js

### Amplify APIs

Authentication

### Amplify Categories

auth

### Environment information

<details>

```
# Put output below this line
System:
    OS: macOS 12.3.1
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 1.05 GB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.13.0 - /usr/local/bin/node
    npm: 8.9.0 - /usr/local/bin/npm
    Watchman: 2021.06.07.00 - /usr/local/bin/watchman
  Browsers:
    Brave Browser: 101.1.38.119
    Firefox: 88.0.1
    Safari: 15.4
  npmPackages:
    @aws-amplify/ui-react: ^2.11.0 => 2.11.0 
    @aws-amplify/ui-react-internal:  undefined ()
    @aws-amplify/ui-react-legacy:  undefined ()
    @testing-library/jest-dom: ^5.16.4 => 5.16.4 
    @testing-library/react: ^13.2.0 => 13.2.0 
    @testing-library/user-event: ^13.5.0 => 13.5.0 
    aws-amplify: ^4.3.17 => 4.3.17 
    react: ^18.1.0 => 18.1.0 (17.0.2)
    react-dom: ^18.1.0 => 18.1.0 
    react-scripts: 5.0.1 => 5.0.1 
    web-vitals: ^2.1.4 => 2.1.4 
  npmGlobalPackages:
    @aws-amplify/cli: 8.3.0
    aws-cdk: 1.141.0
    corepack: 0.10.0
    expo-cli: 5.4.6
    gatsby-cli: 2.12.80
    gatsby: 2.24.67
    npm: 8.9.0
    serverless: 2.38.0
    typescript: 4.4.3
    vercel: 23.1.2

```

</details>

### Describe the bug

During a demo I presented on the AWS Twitch stream, I was unable to get a signed in user to work. 

When on version `4.3.23` of the `aws-amplify` JS library and version `2.18.2` of the `@aws-amplify/ui-react` library, I get an "Auth is not properly configured error". This occurs when using the `withAuthenticator` component and loading a page.

### Expected behavior

When visiting an authenticated page, I should be able to sign in

### Reproduction steps

1. amplify init
2. amplify add auth
3. install `aws-amplify` and `@aws-amplify/ui-react`
4. configure Amplify with `Amplify.configure(awsConfig)`
5. wrap page with `withAuthenticator` component
6. Try to login

### Code Snippet

```javascript
// Put your code below this line.

```

### Log output

<details>

```
// Put your logs below this line


```

</details>

### Additional configuration

I pulled up an old application, used the following versions and it worked as expected:
`@aws-amplify/ui-react@2.11.0`
`aws-amplify@4.3.17`

### Additional information and screenshots

I followed the [troubleshooting guide ](https://docs.amplify.aws/lib/troubleshooting/upgrading/q/platform/js/) to check for duplicates and upgrade accordingly, but that did not work.

**Comment by mtliendo** [comment]
update: I upgraded to the latest version of the UI library and I'm able to sign in so long as I have the older version of the `aws-amplify` library.

**Comment by chrisbonifacio** [comment]
Hi @mtliendo 👋 thanks for raising this issue! I was not able to reproduce the issue with the versions mentioned and the `withAuthenticator` component. Are you still able to reproduce it?

my package.json

```js
  "dependencies": {
    "@aws-amplify/ui-react": "2.18.2",
    "aws-amplify": "4.3.23",
    "next": "12.1.6",
    "react": "18.1.0",
    "react-dom": "18.1.0"
  },
  ```
  
 index.js
 ```js
 import { Auth } from "aws-amplify";
import { withAuthenticator } from "@aws-amplify/ui-react";

import "@aws-amplify/ui-react/styles.css";

const Home = () => {
  return (
    <div>
      <h1>Welcome</h1>
      <button
        onClick={async () => {
          await Auth.signOut();
        }}
      >
        Sign Out
      </button>
    </div>
  );
};

export default withAuthenticator(Home);
 ```

**Comment by elorzafe** [comment]
@mtliendo we published the latest tag of `aws-amplify` to `4.3.24`, can you confirm the issue is solve on your side?

Thanks!

**Comment by elorzafe** [comment]
We had to change `aws-amplify@latest` to `aws-amplify@4.3.23` because of npm outage. Now is pointing to `4.3.24`
